grails-app/views/stockMovement/addComment.gsp

```diff
@@ -69,5 +92,13 @@
 			</div>
 		</div>
 	</div>
+<script type="text/javascript">
+	$(document).ready(function() {
+		const commentInput = $("form [name='comment']")
+		if(${isRequestRejected}) {
```

Comment:
```suggestion
		if (${isRequestRejected}) {
```
😅

Reply:
Is there a better way to do this without using GSP inside javascript? For example, using a hidden input field? Both are hacky but I love mixing languages less than hidden input fields.

Reply:
I completely removed this now and set **required** attribute on the comment field since I believe that it should be required either way, if it is a regular comment or a mandatory comment when rejecting.